LICENSE

```diff
@@ -1,19 +1,19 @@
-Copyright (c) 2012-2015 Adobe Systems Incorporated. All rights reserved.
- 
+Copyright (c) 2012-2016 Adobe Systems Incorporated. All rights reserved.
+
 Permission is hereby granted, free of charge, to any person obtaining a
```

Comment:
Permission is hereby granted, with fee and with charge when using and copy of this software with restriction and associated documentation files (the "Software"),